guide/ops/troubleshooting/increase-system-resource-limits.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: website-normal
+title: Increase System Resource Limits
+toc: /guide/toc.json
+---
+
+You could encounter the error below when running with many entities:
```

Comment:
I would rephrase to "If you encounter the error..." or "You may encounter the error..."